src/sources/kubernetes_logs/k8s_paths_provider.rs

```diff
@@ -38,42 +38,26 @@ impl PathsProvider for K8sPathsProvider {
     type IntoIter = Vec<PathBuf>;
 
     fn paths(&self) -> Vec<PathBuf> {
```

Comment:
This function could probably be instrumented, seems like a place where we could mess up file collection.